router/core/operation_processor.go

```diff
@@ -278,6 +280,14 @@ func (o *OperationKit) UnmarshalOperationFromURL(url *url.URL) error {
 	return o.unmarshalOperation()
 }
 
+func GetOperationNameTrimmed(operationName string, limit int) (bool, string) {
+	if limit <= 0 || len(operationName) <= limit {
+		return false, operationName
+	}
+	operationName = operationName[:limit]
+	return true, operationName
+}
```

Comment:
Nit: Can we do something like take a hash of the full operation name and append it to the truncated one? This way we don't potentially cut off the differentiator and make two operations have the same name?